spec/fixtures/authorities/linked_data/lod_full_config.json

```diff
@@ -150,7 +149,7 @@
           "group_id": "dates",
           "property_label_i18n": "qa.linked_data.authority.locnames_ld4l_cache.birth_date",
           "property_label_default": "Birth",
-          "ldpath": "madsrdf:identifiesRWO/madsrdf:birthDate/schema:label",
+          "ldpath": "madsrdf:identifiesRWO/madsrdf:birthDate/rdfs:label",
```

Comment:
ldpath predefine RDF Schema using rdfs as the namespace.  Updated the config to follow this precedence.